### DISCUSSION TITLE
Send webhooks when an user is created, updated or deleted

### DISCUSSION THREAD

**Discussion by johackim** [directus/directus#8398] [Feature Requests]
It's possible to create webhooks for collections but not for users.

I curently need to create a 'users.create' hook to do that.

Am I the only one to want this feature ? Send a webhook when an user is created is not a common use case ?

**Comment by rijkvanzanten** [comment]
> Am I the only one to want this feature ? Send a webhook when an user is created is not a common use case ?

Apparently so! 

Adding this might be as easy as adding 

```yaml
options:
  includeSystem: true
```

to the following:

https://github.com/directus/directus/blob/fc6b44441c3a283d9acc0203767f73e8444c9288/api/src/database/system-data/fields/webhooks.yaml#L109-L118

Wanna give that a try in a PR @johackim? 🙂

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
